src/app/organisms/emoji-board/EmojiBoardOpener.jsx

```diff
@@ -25,7 +27,10 @@ function EmojiBoardOpener() {
 
   function afterEmojiBoardToggle(isVisible) {
     isEmojiBoardVisible = isVisible;
-    if (!isVisible) {
+    if (isVisible) {
+      searchRef.current.value = '';
+      if (!settings.isTouchScreenDevice) searchRef.current.focus();
+    } else {
```

Comment:
autoFocus will not work in EmojiBoard so we can have it this way. also current method only clears input and doesn't not reset search results.